README.md

```diff
@@ -0,0 +1,46 @@
+# PayPal v6 Web SDK Sample Integration
+
+This repository contains sample integrations for PayPal's v6 Web SDK. Learn how to:
+
+- make one-time payments with different payment methods like PayPal and Venmo
+- save payments methods for future transactions (coming soon)
+- add Pay Later messaging (coming soon)
+
+## Before You Code
+
+1. **Setup a PayPal Account**
+
+   To get started, you'll need a developer, personal, or business account.
+
+   [Sign Up](https://www.paypal.com/signin/client?flow=provisionUser) or [Log In](https://www.paypal.com/signin?returnUri=https%253A%252F%252Fdeveloper.paypal.com%252Fdashboard&intent=developer)
+
+   You'll then need to visit the [Developer Dashboard](https://developer.paypal.com/dashboard/) to obtain credentials and to make sandbox accounts.
+
+2. **Create an Application**
+
+   Once you've setup a PayPal account, you'll need to obtain a **Client ID** and **Secret**. [Create a sandbox application](https://developer.paypal.com/dashboard/applications/sandbox/create).
```

Comment:
I think the Client ID and Secret could be linked more to creating the application.
> Once you've setup a PayPal account, you'll need to obtain a **Client ID** and **Secret** by [creating a sandbox application](https://developer.paypal.com/dashboard/applications/sandbox/create).

or maybe:

> Once you've setup a PayPal account, [create a sandbox application](https://developer.paypal.com/dashboard/applications/sandbox/create) to obtain a **Client ID** and **Secret**.

Reply:
https://github.com/paypal-examples/v6-web-sdk-sample-integration/pull/19